controllers/sr_controller/sr_controller.py

```diff
@@ -2,6 +2,7 @@
 import sys
 import subprocess
```

Comment:
Are we using subprocess anymore in this?

Reply:
I wondered this too. We are -- as part of the handling around getting the version of the code being run.